Directory.Build.props

```
@@ -1,6 +1,7 @@
 <?xml version="1.0" encoding="utf-8"?>
 <Project>
   <Import Project="Sdk.props" Sdk="Microsoft.DotNet.Arcade.Sdk" />
+  <Import Project="$(RepoRoot)eng/Compilers.props" />
```

nit: I think there may be a prop defined by arcade for the eng directory.

---

Oh right: RepositoryEngineeringDir

```suggestion
  <Import Project="$(RepositoryEngineeringDir)Compilers.props" />
```